features/filesystem/fat/FATFileSystem.cpp

```diff
@@ -442,6 +442,34 @@ int FATFileSystem::stat(const char *path, struct stat *st) {
     return 0;
 }
 
+int FATFileSystem::statvfs(const char *path, struct statvfs *buf) {
```

Comment:
`{` new line

Reply:
@deepikabhavnani was being consistent with previous style in this file. I added f1a9815 to correct the file but we shouldn't block a pr based issues that predate the pr.